indico/modules/events/reminders/forms.py

```diff
@@ -85,8 +85,8 @@ def _set_email_choices(self):
         # Current email to avoid destructive modifications
         emails.setdefault(self.reply_to_address.object_data, self.reply_to_address.object_data)
         # Sanitize and format emails
-        emails = {to_unicode(email.strip().lower()): '{} <{}>'.format(name, email)
-                  for email, name in emails.iteritems()
+        emails = {email.strip().lower(): '{} <{}>'.format(name, email)
+                  for email, name in {to_unicode(name): to_unicode(email) for name, email in emails.iteritems()}.iteritems()
                   if email and email.strip()}
```

Comment:
`to_unicode(email).strip()` would probably be enough here. converting twice is not that expensive